src/main/resources/bigquery-to-parquet-command-spec.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why do we need this file?

Reply:
This file will be used by jib to copy the correct class to the container.